hw/top_earlgrey/dv/Makefile

```diff
@@ -26,7 +26,9 @@ RAL_TOOL_OPTS   += --top
 
 # Common build options.
 # Enable the RISC-V Formal Interface (RVFI) for the ibex tracer.
-BUILD_OPTS      += +define+RVFI=1
+ifeq (${TRACE},1)
+  BUILD_OPTS      += +define+RVFI=1
+endif
```

Comment:
@sriyerg I've added this to remove dumping the trace file by default.

Reply:
Can you rename it to IBEX_TRACER instead? Thanks for htis. I will add something similar to this in the dvsim flow.

Reply:
OK I will create a follow-up PR.